crates/livesplit-hotkey/src/macos/mod.rs

```diff
@@ -488,6 +502,7 @@ unsafe extern "C" fn callback(
         .get_mut(&key_code.with_modifiers(modifiers))
     {
         callback();
+        CGEventSetType(event, EventType::NULL);
```

Comment:
This seems to be the main change, considering we actually (accidentally?) had `DEFAULT_TOP` in the non-consuming implementation. Apple's documentation seems very unclear that this would cause the key to be consumed. Does this not need any `if` that guards it to only one implementation? No matter what, I'd like some comment explaining what's going on here.

Reply:
I've seen some implementations do this in the wild, but after reviewing [this page](https://developer.apple.com/documentation/coregraphics/cgeventtapcallback) I realized it isn't necessary and we can just return a null pointer. That clearly says that the return value will be ignored in the passive listener case. Yes `DEFAULT_TAP` could have been `LISTEN_ONLY` before, but now the difference will become load bearing. I added a comment.